apps/examples/esp32_wifi_demo/station.c

```diff
@@ -0,0 +1,119 @@
+/******************************************************************
+ *
+ * Copyright 2019 Samsung Electronics All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ ******************************************************************/
+
+/* WiFi station Example
+   This example code is in the Public Domain (or CC0 licensed, at your option.)
+   Unless required by applicable law or agreed to in writing, this
+   software is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR
+   CONDITIONS OF ANY KIND, either express or implied.
+*/
+
+#include "esp_wifi.h"
+#include "esp_log.h"
+#include "esp_event_loop.h"
+#include "esp_wifi_internal.h"
+
+/*
+ The examples use WiFi configuration that you can set via 'make menuconfig'.
+ If you'd rather not, just change the below entries to strings with
+ the config you want - ie #define EXAMPLE_WIFI_SSID "mywifissid"
+*/
+
+#define DEFAULT_SSID CONFIG_ESP_WIFI_SSID
```

Comment:
To use CONFIG_XX, please include <tinyara/config.h>

Reply:
tks, fixed,  refer to other app,  using config item directly in sample code.